vercel-api/docs/LOAD_TESTING_GUIDE.md

```
@@ -0,0 +1,526 @@
+   export TEST_API_URL="http://localhost:3000"
+   export TEST_WORDPRESS_URL="https://your-wordpress-site.com"
+   export TEST_API_KEY="your-test-api-key"
+   ```
+
+2. **Optional k6 Installation**:
+   ```bash
+   # macOS
+   brew install k6
+   
+   # Linux (Ubuntu/Debian)
+   sudo apt-get update
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)